Registration forms sub-chapter (#3160)
alan-turing-institute/the-turing-way: book/website/collaboration/organising-conference/reg-forms.md

(cl-organising-conferences-reg-forms)=

> # Registration forms

Reviewer: ```suggestion
# Registration Forms
```

(cl-organising-conferences-reg-forms-host)=

> ## Where will you host your registration form?

Reviewer: ```suggestion
## Where will you host your Registration Form?
```

Under GDPR (https://en.wikipedia.org/wiki/General_Data_Protection_Regulation), you are required to collect the minimum amount of data and especially when it comes to personal data. 
So think carefully about what you really do require for the purpose of the event and also the outcomes of the event such as a report. 

You might not think you are collecting sensitive or special category data in a registration form but this can be disclosed by attendee. 
- Questions about accessibility requirements can give answers that include health information.
- Questions about dietary requirements can receive responses that record the religious belief of attendees and also health information such as allergies. 

You must clearly explain why you’re asking for sensitive information and how you will handle this data.
This includes clearly stating what you will do with the data afterwards - will this be to inform you on equality, diversity and inclusion, will you use it to write a report?

Organisations usually have a privacy statement that can be used for this purpose. 
An example of a privacy notice can be found on the The Alan Turing Institute website: https://www.turing.ac.uk/privacy-policy 

> (cl-organising-conferences-reg-forms=data-must)=

Author: ```suggestion
(cl-organising-conferences-reg-forms-data-must)=
```

This can include age (or banded age categories), ethnic origin, gender. 
This data is considered sensitive so should be restricted access and the colleciton should be minimised.
However, this data is important for monitoring the equality, diversity and inclusion monitoring of your events and therefore important to track whether you are being inclusive.  

(cl-organising-conferences-reg-forms-tips)=
## General tips
Sequence your questions logically after each other.

If you ask an optional question, make it clear that it’s optional.

Do not rely on colour to communicate your questions.

Provide clear explanations for complicated questions to guide people towards choosing the correct answer/option.

Provide contact information of the committee member who can be contacted regarding any errors or questions related to registration.

Clearly state the next steps upon clicking the submit button.

Link the registration form clearly on the webpage and advertise across different community platforms.

(cl-organising-conferences-reg-forms-template)=
## Template registration form

We have made a template of all the different questions you might want to include on your registraion form. 

> You can see the template in our community handbook's tenplate collection. 

Author: ```suggestion
You can see and used our registration form template in using this link 
```
